azure/azure-how-to/instances/create-a-golden-image-pipeline.rst

```diff
@@ -0,0 +1,274 @@
+Create an Ubuntu golden image pipeline
+======================================
+
+This guide will provide instructions for using GitHub Actions to create a pipeline for building Ubuntu "golden" images with the Azure Image Builder (AIB). This guide focuses on automating the use of Azure Image Builder. If you want more information on how to use Azure Image Builder see the :doc:`Create a Pro golden image <create-pro-fips-golden-image>` guide which explains the individual steps.
```

Comment:
Can we add an introductory statement here about the possible use(s) of such a pipeline? Say to get updated versions of a golden image.